src/app/auth/features/user-manager-detail/user-manager-detail.component.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * User Manager Detail component styles
+ * @copyright The GHGA Authors
+ * @license Apache-2.0
+ */
+
+@use '@angular/material' as mat;
+
+:host {
+  display: block;
+  padding-bottom: 1em;
+  background-color: var(--mat-sys-surface-container-low);
```

Comment:
Not sure I like the separate background colour. Maybe make the borders rounded at least, and a bit more contrasting?

Reply:
See above.